components/org.wso2.carbon.identity.branding.preference.management.core/src/main/java/org/wso2/carbon/identity/branding/preference/management/core/UIBrandingPreferenceResolver.java

```diff
@@ -39,13 +39,27 @@ public interface UIBrandingPreferenceResolver {
      */
     BrandingPreference resolveBranding(String type, String name, String locale) throws BrandingPreferenceMgtException;
 
+    /**
+     * This method is used to clear the branding preference resolver caches, down
+     * the organization hierarchy.
+     *
+     * @param type                Branding preference type.
+     * @param name                Branding preference name.
+     * @param currentTenantDomain Tenant domain where the cache needs to be cleared.
+     * @throws BrandingPreferenceMgtException if any error occurred.
+     */
+    void clearBrandingResolverCacheHierarchy(String type, String name, String currentTenantDomain)
```

Comment:
Make it default.

Reply:
Addressed with https://github.com/wso2-extensions/identity-branding-preference-management/pull/40/commits/05fa6da03cb482ecbd514d7030744011bad7a98e